roxie/ccd/ccdmain.cpp

```diff
@@ -770,10 +772,15 @@ int STARTQUERY_API start_query(int argc, const char *argv[])
             removeLog();
         if (topology->hasProp("@logfile"))
         {
+#ifndef _CONTAINERIZED
             Owned<IComponentLogFileCreator> lf = createComponentLogFileCreator(topology, "roxie");
             lf->setMaxDetail(TopDetail);
             lf->beginLogging();
             logDirectory.set(lf->queryLogDir());
+#else
+            setupContainerizedLogMsgHandlerFromConfig();
```

Comment:
don't think it makes sense for this to be under if (topology->hasProp("@logfile")) now.
The #ifndef _CONTAINERIZED should be above that.

There are other potentially interesting properties here related to logging in general, like "logQueueLen" and "logQueueDrop".
Should also be properties of the generic logging: config, and picked up in setupContainerizedLogMsgHandlerFromConfig()